utils/config-check.ts

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
same as above, already in `src/utils` and I can't tell if anything was actually changed here.